internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json

```diff
@@ -242,913 +242,953 @@
-              "value": 766573
+              "date": "2020",
+              "value": 764442
             },
             "geoId/06083": {
-              "date": "2019",
-              "value": 446499
+              "date": "2020",
+              "value": 448229
             },
             "geoId/06085": {
-              "date": "2019",
-              "value": 1927852
+              "date": "2020",
+              "value": 1936259
             },
             "geoId/06087": {
-              "date": "2019",
-              "value": 273213
+              "date": "2020",
+              "value": 270861
             },
             "geoId/06089": {
-              "date": "2019",
-              "value": 180080
+              "date": "2020",
+              "value": 182155
+            },
+            "geoId/06091": {
+              "date": "2020",
+              "value": 3236
+            },
+            "geoId/06093": {
+              "date": "2020",
+              "value": 44076
             },
             "geoId/06095": {
-              "date": "2019",
-              "value": 447643
+              "date": "2020",
+              "value": 453491
             },
             "geoId/06097": {
-              "date": "2019",
-              "value": 494336
+              "date": "2020",
+              "value": 488863
             },
             "geoId/06099": {
-              "date": "2019",
-              "value": 550660
+              "date": "2020",
+              "value": 552878
             },
             "geoId/06101": {
-              "date": "2019",
-              "value": 96971
+              "date": "2020",
+              "value": 99633
             },
             "geoId/06103": {
-              "date": "2019",
-              "value": 65084
+              "date": "2020",
+              "value": 65829
+            },
+            "geoId/06105": {
+              "date": "2020",
+              "value": 16112
             },
             "geoId/06107": {
-              "date": "2019",
-              "value": 466195
+              "date": "2020",
+              "value": 473117
+            },
+            "geoId/06109": {
+              "date": "2020",
+              "value": 55620
             },
             "geoId/06111": {
-              "date": "2019",
-              "value": 846006
+              "date": "2020",
+              "value": 843843
             },
             "geoId/06113": {
-              "date": "2019",
-              "value": 220500
+              "date": "2020",
+              "value": 216403
             },
             "geoId/06115": {
-              "date": "2019",
-              "value": 78668
+              "date": "2020",
+              "value": 81575
             }
           },
-          "meta_hash": 2071679530
+          "meta_hash": 2939218655
         },
         {
           "stat": {
             "geoId/06001": {
-              "date": "2019",
-              "value": 1656754
+              "date": "2011",
```

Comment:
Puzzled why the date is older now.

Reply:
Ohh this is bad json diffs I think :(.   Perhaps we should sort or something.

Reply:
Sorted by sources now